uaclient/cli.py

```diff
@@ -397,6 +397,13 @@ def attach_parser(parser):
             " on the cli"
         ),
     )
+    parser.add_argument(
```

Comment:
I am not creating the `--assume-yes` flag here because we don't prompt the users during attach. But I can add it just for consistency sake

Reply:
/meis fine as well